### PR TITLE
Rework GetConsumer

### PR DESCRIPTION
**Pull request by Cheephot**

### CODE REVIEW

**Review by VLiamin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Review by djino97 — APPROVED** [review]

File: src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 1]
```diff
@@ -4,17 +4,8 @@ namespace LT.DigitalOffice.ImageService.Models.Dto.Configuration
...
-        public string CreateImagesMessageEndpoint { get; set; }
-        public string GetImagesMessageEndpoint { get; set; }
-        public string DeleteImagesMessageEndpoint { get; set; }
-        public string CreateImagesNewsEndpoint { get; set; }
-        public string GetImagesNewsEndpoint { get; set; }
-        public string DeleteImagesNewsEndpoint { get; set; }
-        public string CreateImagesUserEndpoint { get; set; }
-        public string GetImagesUserEndpoint { get; set; }
-        public string DeleteImagesUserEndpoint { get; set; }
+        public string CreateImagesEndpoint { get; set; }
```
Comment (MariyaSin) [inline src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs:7]: rename to remove

File: src/ImageService/appsettings.json [hunk 2]
```diff
@@ -48,18 +48,9 @@
...
-    "DeleteImagesNewsEndpoint": "ImageService_DeleteImagesNews",
-    "GetImagesMessageEndpoint": "ImageService_GetImagesMessage",
-    "CreateImagesMessageEndpoint": "ImageService_CreateImagesMessage",
-    "DeleteImagesMessageEndpoint": "ImageService_DeleteImagesMessage",
-    "GetImagesProjectEndpoint": "ImageService_GetImagesProject",
-    "CreateImagesProjectEndpoint": "ImageService_CreateImagesProject",
-    "DeleteImagesProjectEndpoint": "ImageService_DeleteImagesProject"
+    "GetImagesEndpoint": "ImageService_GetImages",
+    "CreateImagesEndpoint": "ImageService_CreateImages",
+    "DeleteImagesUEndpoint": "ImageService_DeleteImages"
```
Comment (MariyaSin) [inline src/ImageService/appsettings.json:53]: remove

**Review by Glunky — APPROVED** [review]

File: src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs (hunk 1 above)
Comment (Cheephot) [reply src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs:7]: done

File: src/ImageService/appsettings.json (hunk 2 above)
Comment (Cheephot) [reply src/ImageService/appsettings.json:53]: done

**Pull request merged**